### PR TITLE
[WIP] Jenkins pipelines taks

### PR DESCRIPTION
**Pull request by danpaldev**

### CODE REVIEW

**Comment by danpaldev** [comment]
For the moment, the ```job_configs/thanks_plugin.groovy``` file is the only working file. 

Please ignore ```job_configs/first_job.groovy```, ``` jobs/first_job.groovy``` and ```jobs/first_job.jenkinsfile```. I created these files just to understand how everything wraps-up and don't have any significance for the task.

Please note that I use environment variables in the docker-compose file, ```GITHUB_USER``` and ```GITHUB_PASS```, to load my credentials into jenkins container. For obvious reasons, I didn't track/commit the ```.env``` file that contains my github credentials.

File: .env [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Git repo url for the seed jobs (which uses the groovy job dsl plugin)
-SEED_JOBS_URL=https://github.com/learningdevops-makvaz-com/ci/
+SEED_JOBS_URL=https://github.com/danpaldev/phase02_task04.git
 
 # Admin credentials
 ADMIN_USERNAME=admin
-ADMIN_PASSWORD=changeme
+ADMIN_PASSWORD=password
```
Comment (korney4eg) [inline .env:6]: :P

File: README.md [hunk 2]
```diff
@@ -1,3 +1,105 @@
-# Task 04 on Phase 02.
+## What is Jenkins Pipeline?
```
Comment (korney4eg) [inline README.md:1]: great README

**Review by korney4eg — COMMENTED** [review]
So far looks good

**Pull request closed without merging**